LondonBritishLibrary/orient/BLorient735.xml

```diff
@@ -52,48 +183,47 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">241</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>305</height>
-                                        <width>260</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>12</height>
+                                        <width>10.25</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
                             </supportDesc>
-                            
                             <layoutDesc>
                                 <layout columns="3" writtenLines="33" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
                         <handDesc>
-                            
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is written in a fair hand,  of the <date>18th century</date>.
                                 </desc>
-                                
                                 <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
-                                
                             </handNote>
-                            
-                        </handDesc>   
+                        </handDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                                    <locus target="#3r"/>
+                                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                        <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                <q xml:lang="gez"><title ref="LIT2616Zenaes">አፈ፡ ወርቅ፡</title> ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
```

Comment:
I don't see a reason for the identification with LIT2616Zenaes in the manuscript

Reply:
You are right, I have overlooked that